src/main/java/seedu/address/commons/util/JsonUtil.java

```diff
@@ -37,7 +37,8 @@ public class JsonUtil {
             .setVisibility(PropertyAccessor.FIELD, JsonAutoDetect.Visibility.ANY)
             .registerModule(new SimpleModule("SimpleModule")
                     .addSerializer(Level.class, new ToStringSerializer())
-                    .addDeserializer(Level.class, new LevelDeserializer(Level.class)));
+                    .addDeserializer(Level.class, new LevelDeserializer(Level.class))
+                    .addSerializer(Path.class, new ToStringSerializer()));
```

Comment:
Good that this was added, but will there be a need to add a deserialiser for the path.class as well?
```suggestion
                    .addSerializer(Path.class, new ToStringSerializer())
                    .addDeserializer(Path.class, new LevelDeserializer(Path.class)));
```